### PR TITLE
docs: add RedGradient to showcase

### PR DESCRIPTION
**Pull request by redcamel**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [x] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [x] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

### Test links

Deploy preview: https://deploy-preview-_____--docusaurus-2.netlify.app/

### CODE REVIEW

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -1700,6 +1700,14 @@ const Users: User[] = [
     source: null,
     tags: [],
   },
+  {
+    title: 'RedGradient',
+    description: 'CSS Gradient Editor',
+    preview: require('./showcase/red-gradient.png'),
+    website: 'https://doc.red-gradient.com/',
+    source: 'https://github.com/redcamel/RedGradient',
```
Comment (Josh-Cena) [inline website/src/data/users.tsx:1708]: Is your **site** open-source? I failed to find any Docusaurus source here.

Comment (redcamel) [reply website/src/data/users.tsx:1708]: https://github.com/redcamel/RedGradientDoc 인데 잘못기재한것 같습니다
다시 업로드 할까요?

Comment (redcamel) [reply website/src/data/users.tsx]: https://github.com/redcamel/RedGradient ===> https://github.com/redcamel/RedGradientDoc changed.

File: website/src/data/users.tsx [hunk 2]
```diff
@@ -48,7 +48,7 @@ import {sortBy} from '@site/src/utils/jsUtils';
  */
 
 // LIST OF AVAILABLE TAGS
-// Available tags to assign to your site
+// Available tags to assign to your site
```
Comment (slorber) [inline website/src/data/users.tsx:51]:
```suggestion
// Available tags to assign to your site
```

**Comment by slorber** [comment]
Thanks 👍

**Pull request merged**